src/cadence/mainnet/getNFTs.cdc

```diff
@@ -2429,8 +2433,8 @@ pub fun getAmericanAirlinesNFT(owner: PublicAccount, id: UInt64): NFTData? {
             NFTMedia(uri: setMeta!["preview"], mimetype: "image")
         ],
         metadata: {
-            "edition": nft!.editionNum.toString(),
-            "max_editions": nftEditions!.toString(),
+            "editionNumber": nft!.editionNum.toString(),
```

Comment:
Did you intend to make this rename change?

Reply:
Yes the name for this was changed throughout the file based on @albeethekid recommendation to align with the Blocto naming convention